src/components/calculator.js

```diff
@@ -1,108 +1,100 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import Blocks from './Blocks';
 import calculate from '../logic/calculate';
 
-export default class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-  }
+const Calculator = () => {
+  const [calcState, setCalcState] = useState({});
 
-  handleCalculate = (label) => {
-    const { total, next, operation } = this.state;
+  const handleCalculate = (label) => {
+    const { total, next, operation } = calcState;
     const newObj = calculate({ total, next, operation }, label);
-    this.setState({
+    setCalcState({
       total: newObj.total,
       operation: newObj.operation,
       next: newObj.next,
     });
   };
```

Comment:
- [ ] [REQUIRED] Please make sure that the next operand is displayed on the screen when users enter it.
![20220604_102106](https://user-images.githubusercontent.com/58552748/171989421-fbd3940c-4a80-49af-ba30-07368d802bce.gif)